src/coreclr/jit/gentree.cpp

```diff
@@ -12093,7 +12093,14 @@ void Compiler::gtDispTree(GenTree*     tree,
                 case NI_System_Runtime_CompilerServices_RuntimeHelpers_IsKnownConstant:
                     printf(" isKnownConst");
                     break;
-
+#if defined(FEATURE_SIMD)
+                case NI_SIMD_UpperRestore:
+                    printf(" simd_upperRestr");
+                    break;
+                case NI_SIMD_UpperSave:
+                    printf(" simd_upperSave");
+                    break;
+#endif // FEATURE_SIMD
                 default:
                     unreached();
```

Comment:
Given this is debug only and just impacts JITDUMP, can we do something like `assert` or dump the "raw id" as a fallback?

That is, we could just `printf(" unknown: %d", static_cast<uint16_t>(id))` or something, right?

Reply:
Doing so at least gives usable information and doesn't make getting dumps impossible